tests/python/testing.py

```diff
@@ -209,6 +211,29 @@ def get_sparse():
     return X, y
 
 
+@memory.cache
+def get_mq2008(dpath):
+    from sklearn.datasets import load_svmlight_files
```

Comment:
What is the runtime of these new tests? Some time ago, I noticed a large amount of time is consumed by ranking tests.

Reply:
[2021-01-07T13:18:15.483Z] 2.04s teardown python/test_with_dask.py::test_dask_ranking

https://xgboost-ci.net/blue/organizations/jenkins/xgboost/detail/PR-6576/8/pipeline/206